kinder/doc/prepare-for-tests.md

```diff
@@ -34,18 +33,15 @@ Build a base-image (or download one from docker hub)
 kinder build base-image --image kindest/base:latest
 ```
 
```

Comment:
for this section, i think we can remove the sentence on line 29:
> For building a node image you can refer to kind documentation; below a short recap of necessary steps:

that is because kinder drifted away from kind with the node-image-variant command.